README.md

```diff
@@ -4,9 +4,9 @@ The Inclusive Coding Environment is a new educational coding tool designed for l
 
 ![inclusive coding environment UI](./inclusive-coding-env.png)
 
-To try the latest development version of the Inclusive Coding Environment, visit our [development build](https://c2lc-dev.web.app/).
+To try the latest development version of the Inclusive Coding Environment, visit our [development build](https://develop-0-6--c2lc-build.netlify.app).
```

Comment:
I think at this time, we can just point to the production version of the application:

    To try the latest version of the Inclusive Coding Environment,
    visit our [coding environment online](https://build.codelearncreate.org/).

This will save us from having to update the development link for each new version.